VTEX - Promotions & Taxes API.json

```diff
@@ -1774,18 +1794,38 @@
                                             },
                                             "listSku1BuyTogether": {
                                                 "type": "array",
-                                                "description": "SKU first list for the promotion **Buy Together**.",
+                                                "description": "Array of objects, each containing ID and Name of an SKU to be added in the first list of the **Buy Together** promotion. Learn more about this type of promotion in the [Creating a Buy Together promotion](https://help.vtex.com/en/tutorial/buy-together--tutorials_323) documentation.",
```

Comment:
O que significa "the first list of the **Buy Together** promotion"? (se for alterar aqui para explicar melhor, precisa alterar em todos os lugares onde essa descrição aparece)

Reply:
Trata-se de um array que contém uma lista de SKU, adicionado na primeira lista da promoção compre junto.